### PR TITLE
Project Reoganisation

### PR DESCRIPTION
**Pull request by Kickball**
Overhaul of project layout.

* Move authors to AUTHORS.
* Move contributing guidelines to CONTRIBUTING.md.
* Add PULL_REQUEST_TEMPLATE.

Remake of #623.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Contributing
+
+Please join us on [chat](https://chat.awesome-selfhosted.com) to clarify any questions.
+
+ * The **syntax** for links is ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language` ``. All projects should have their **license** and main server-side **language**/platform/requirement listed, and a description at most 250 characters long.
```
Comment (AndyR207) [inline .github/CONTRIBUTING.md]: [Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Example](http://url.of/example), [Source Code](http://url.of/source/code)) `License` `Language`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Thank you for taking the time to work on a PR for Awesome-selfhosted!
+
+To ensure your PR is dealt with swiftly please check the following:
+
+- [ ] Your submissions are formatted according to the guidelines. 
+        
+    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
```
Comment (AndyR207) [inline .github/PULL_REQUEST_TEMPLATE.md]: [Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Example](http://url.to/example), [Source Code](http://url.of/source/code)) `License` `Language`

**Comment by Kickball** [comment]
Fixed the formatting guidelines.

**Comment by nodiscc** [comment]
Everything looks good, thank you

**Comment by n8225** [comment]
Looks good

**Comment by Kickball** [comment]
Just added some new commits to reverse ae0640e and db349f6 as both @AndyR207 and myself forgot about the changes made removing example in #439.

**Pull request merged**